packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -1701,6 +1703,30 @@ export function getRenderLanes(): Lanes {
   return renderLanes;
 }
 
+function resetWorkInProgressStack() {
+  if (workInProgress == null) return;
+  let interruptedWork;
```

Comment:
This function should set `workInProgress` to `null`, too.

Reply:
Is it because once we unwind the `workInProgress` is no longer useful, so it's better to reset it here?

Reply:
Yeah `workInProgress` represents the next "stack frame" so once we unwind the stack it should be reset